packages/payments-plugin/e2e/mollie-payment.e2e-spec.ts

```diff
@@ -156,334 +157,370 @@ describe('Mollie payments', () => {
         expect(customers).toHaveLength(2);
     });
 
-    it('Should prepare an order', async () => {
-        await shopClient.asUserWithCredentials(customers[0].emailAddress, 'test');
-        const { addItemToOrder } = await shopClient.query<AddItemToOrder.Mutation, AddItemToOrder.Variables>(
-            ADD_ITEM_TO_ORDER,
-            {
-                productVariantId: 'T_5',
-                quantity: 10,
-            },
-        );
-        order = addItemToOrder as TestOrderFragmentFragment;
-        // Add surcharge
-        const ctx = new RequestContext({
-            apiType: 'admin',
-            isAuthorized: true,
-            authorizedAsOwnerOnly: false,
-            channel: await server.app.get(ChannelService).getDefaultChannel(),
+    describe('Payment intent creation', () => {
```

Comment:
Sorry for the large diff, but I grouped the testcases for readability: 
![image](https://user-images.githubusercontent.com/6604455/226935545-5530ca16-1f57-45da-890c-3f7b4096afc0.png)